etc/parameters/default.json

```diff
@@ -196,6 +196,12 @@
         "first_half_own_half_away_own_goal": 25.0,
         "first_half_opponent_half_towards_own_goal": 25.0,
         "first_half_opponent_half_away_own_goal": 25.0
+      },
+      "luminance_threshold": {
+        "first_half_own_half_towards_own_goal": 25.0,
+        "first_half_own_half_away_own_goal": 25.0,
+        "first_half_opponent_half_towards_own_goal": 25.0,
+        "first_half_opponent_half_away_own_goal": 25.0
```

Comment:
This needs to be tuned for usual green fields. I guess the value needs to be way higher.

Reply:
Yes, that is very conservative. I would leave it like this and only increase it if the light conditions are really bad.